### ISSUE TITLE
nfs-ganesha 2.7.1, Ubuntu Bionic, missing dependency on liburcu-bp.so.6

### ISSUE THREAD

**Issue by koshatul** [nfs-ganesha/nfs-ganesha-debian#27]
After installing from the PPA the nfs-ganesha service won't start

```
bash[5203]: /usr/bin/ganesha.nfsd: error while loading shared libraries: liburcu-bp.so.6: cannot open shared object file: No such file or directory
systemd[1]: nfs-ganesha.service: Control process exited, code=exited status=127
systemd[1]: nfs-ganesha.service: Failed with result 'exit-code'.
systemd[1]: Failed to start NFS-Ganesha file server.
```

I'm looking for the dependency now, I've only just encountered the error so I'm looking for the package that provides `liburcu-bp.so.6` now.

**Comment by koshatul** [comment]
I'm blind on the package, after installing `liburcu6` from the ubuntu repo it worked.

```
sudo apt-get install -qy liburcu6
```

But still a missing dependency.

**Comment by kalebskeithley** [comment]
stupid pbuilder doesn't automatically determine dependencies from the build or -dev packages like rpm does.
Fixed in the latest 2.7.2 builds
